examples/lighting-app/qpg/src/ZclCallbacks.cpp

```diff
@@ -62,6 +64,70 @@ void emberAfPostAttributeChangeCallback(EndpointId endpoint, ClusterId clusterId
             ChipLogError(Zcl, "wrong length for level: %d", size);
         }
     }
+    else if (clusterId == ZCL_COLOR_CONTROL_CLUSTER_ID)
+    {
+        /* ignore several attributes that are currently not processed */
+        if ((attributeId == ZCL_COLOR_CONTROL_REMAINING_TIME_ATTRIBUTE_ID) ||
+            (attributeId == ZCL_COLOR_CONTROL_ENHANCED_COLOR_MODE_ATTRIBUTE_ID) ||
+            (attributeId == ZCL_COLOR_CONTROL_COLOR_MODE_ATTRIBUTE_ID))
+        {
+            return;
```

Comment:
should we log as not processed? or are these commonly sent but never used and don't want to pollute logs?

Reply:
These are always sent, but currently not processed - I think we should not pollute the logs with them, and just ignore them for now.